pkg/report/formatter.go

```diff
@@ -65,10 +65,10 @@ func (f *Formatter) Parse(origin Origin, text string) (*Formatter, error) {
 	// parse the template. If it fails use the original text and parse again.
 	var normText string
 	switch {
-	case strings.HasPrefix(text, "table "):
+	case strings.HasPrefix(text, "table "): // This could use strings.CutPrefix after we require Go 1.20
 		f.RenderTable = true
 		normText = "{{range .}}" + NormalizeFormat(text) + "{{end -}}"
-		text = "{{range .}}" + text + "{{end -}}"
+		text = "{{range .}}" + text[len("table "):] + "{{end -}}"
```

Comment:
This looks correct, I assume you do not want to hardcode it to 6 for readability? However one might still change the string above without updating this one so I say combine them into one variable.

Reply:
Actually, let me do the methodical thing and backport the future-standard-library function, instead of suggesting that someone else should restructure the code in the future.